build/reindex/bin/gather-logs.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# (C) Copyright IBM Corp. 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+
+set -ex
+
+# Gathers the logs
+package_logs(){
```

Comment:
I wonder if this one should be shared accross the various e2e tests.  Maybe introduce a `build/common` or something?

Reply:
moved.